### ISSUE TITLE
Progress Bar

### ISSUE THREAD

**Issue by Jim-Booth** [DataJuggler/BlazorFileUpload#19]
Is it possible to use the filestream to create an upload progress bar

**Comment by DataJuggler** [comment]
Yes, and the sample I copied had one and I regret not implementing it. I have been doing more 3D animation just so I don't stare code all day and night. I will see if I can find the sample I used when I updated to .Net 5 and this weekend I will have a look at this. 

There are times I think an upload has failed, and right before I give up it finishes. Others might not know to wait.
